source/develop/release-management/features/virt/virt-sparsify.html.md

```diff
@@ -41,6 +41,18 @@ Added "Sparsify" button to Disks subtab of Virtual Machines main tab. This butto
 * Cinder disks cannot be sparsified.
 * Pre-allocated disks cannot be sparsified. `virt-sparsify` works on pre-allocated disks, but its usage is not justified for this case. User chooses pre-allocated option if she needs better performance, because no additional allocation and no fragmentation will occur when the disk is used. Deallocating some clusters as result of sparsification will overturn this advantage. If user wants to save space at cost of some performance decrease, she can select thin-provisioned option.
 * Disks on NFS storage may be sparsified only if NFS version >= 4.2.
+* For an iSCSI volume to be sparsified, support of both host kernel and the actual backing store is required.
+
+   Also it depends on various SCSI flags reported by the device. These are contained in the SCSI "VPD" data. It might be interesting to run `sg_inq` and `sg_vpd` (from `sg3_utils`) on the device:
```

Comment:
Rephrase 'Also it depends' to something like - you can verify this support via various SCSI flags...

Reply:
Done.